reference/v8.json

```diff
@@ -639,14 +699,22 @@
       "function": "piecewise-constant",
       "zoom-function": true,
       "property-function": false,
-      "values": [
-        "none",
-        "both",
-        "width",
-        "height"
-      ],
+      "values": {
+        "none": {
+            "doc": "No scaling is performed."
```

Comment:
"The icon is displayed at its intrinsic aspect ratio." (`icon-size` can still scale it.)